shared/common-adapters/popup-dialog.js.flow

```diff
@@ -6,10 +6,10 @@ export type Props = {
   allowClipBubbling?: boolean,
   fill?: boolean,
   children?: React.Node,
-  styleCover?: Object,
-  styleContainer?: Object,
-  styleClose?: Object,
-  styleClipContainer?: Object,
+  styleCover?: any,
+  styleContainer?: any,
+  styleClose?: any,
+  styleClipContainer?: any,
 }
```

Comment:
just wondering why any is better than Object ?

Reply:
What's acceptable as a style is a very complicated object actually. in practice it can be an object, it can be undefined/null/false, it can be a number (RN Stylesheet.create makes an id that's used) etc so since we don't have a really great type we have to expand it. we could make a correct type but it'll take some time

Reply:
also in RN it can take an array